build/verify-features.ts

```diff
@@ -60,6 +60,10 @@ function findError(filename: string): string | void {
 		return `ERR: ${featureId} should be described better in the readme (at least 20 characters)`;
 	}
 
+	if (featureMeta.screenshot && !/\.(png|jpg|gif)$/.test(featureMeta.screenshot)) {
```

Comment:
Screenshots should not be jpegs

Reply:
Done, but the following three features have a .jpg image:

```
ERR: comment-on-draft-pr-indicator should have a screenshot (png/gif) in the readme
ERR: link-to-github-io should have a screenshot (png/gif) in the readme
ERR: one-click-pr-or-gist should have a screenshot (png/gif) in the readme
```

Reply:
Can you download them and reload them and just change the extension before uploading it.

Reply:
Changed the extension, but the files are still jpegs

Reply:
Converted the files to pngs 🎉